index.html

```diff
@@ -0,0 +1,75 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Oluyemi Portfolio</title>
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+<link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+<link href="https://fonts.googleapis.com/css2?family=Crete+Round&family=Poppins:wght@100;200&family=Roboto:wght@100;200&display=swap" rel="stylesheet">
+    <script src="https://kit.fontawesome.com/746ba03926.js" crossorigin="anonymous"></script>
+    <link rel="stylesheet" href="styles.css">
+</head>
+
+<body>
+    <header>
+        <nav id="navbar">
+            <div id="profileName">
+                <p><strong>Paul, Oluyemi</strong></p>
+            </div>
+            <div id="bars">
+                <div class="bar1"></div>
+                <div class="bar2"></div>
+                <div class="bar3"></div>
+            </div>
+        </nav>
+    </header>
+    <main id="container">
+        <header id="greetText">
+            <h1>Hey There.<br>
+                I'm Oluyemi</h1>
+        </header>
+        <section>
+            <h2 id="jobTitle">I am a Software Developer</h2>
+            <p id="description">I can help you build a product , feature or website Look through
+                some of my work and experience! If you like what you see and
+                have a project you need coded,
+                don’t hestiate to contact me.</p>
+        </section>
+        <ul class="icon">
+            <li><a href="https://twitter.com/OluyemiPaul99"><i class="fa fa-twitter fa-lg"></i> </a></li>
+            <li><a href="https://www.linkedin.com/in/paul-oluyemi-193966ab"><i class="fa fa-linkedin fa-lg"></i> </a>
+            </li>
+            <li><a href="#"><i class="fa fa-medium fa-lg"></i> </a></li>
+            <li><a href="https://github.com/Ol-create"><i class="fa fa-github fa-lg"></i> </a></li>
+            <li><a href="#"><i class="fa fa-angellist fa-lg"></i> </a></li>
+        </ul>
```

Comment:
Great job general design is very good! All background images, margins, and details are really similar to Figma's!

Reply:
Thank you for your effort. I appreciate you.